src/app/app.component.scss

```diff
@@ -0,0 +1,63 @@
+$primaryBtnBgColor: #009ecc;
```

Comment:
you can move these variables to the `_variables.scss` file. Also, you can define them as a part of a map as the following.
```
$app-primary-palette: (
  dialog: #eee,
  50: #fde6e5,
  100: #fac1be,
  200: #f69892,
  300: #f26f66,
);

usage 
.selector {
  color: map-get($app-primary-palette, dialog);
}

Reply:
done